docs/source/schema/directives.md

```diff
@@ -105,18 +105,19 @@ If you want to define a custom schema directive to _transform_ your executable s
 
 ### In subgraphs
 
-> ⚠️ **Important considerations**
->
-> Before you use directives in a federated graph, make sure to consider the following:
->
-> - Custom directives are _not_ included in your graph's composed supergraph schema. The composition process strips all subgraph directives. Only a given subgraph is aware of its own directives.
-> - Because directives are specific to individual subgraphs, it's valid for different subgraphs to define the _same_ directive with _different_ logic. Composition does not detect or warn about such inconsistencies.
-> - If multiple subgraphs can resolve a particular field, each subgraph should almost always apply the exact same set of custom directives (with the exact same accompanying logic) to that field. Otherwise, the behavior of that field might vary depending on _which_ subgraph resolves it.
+Before you use custom directives in a federated graph, make sure to consider the following:
+- If multiple subgraphs can resolve a particular field, each subgraph should almost always apply the exact same set of custom directives (with the exact same definition) to that field. Otherwise, the behavior of that field might vary depending on _which_ subgraph resolves it.
+- Because directives are specific to individual subgraphs, it's technically valid for different subgraphs to define the _same_ directive with _different_ logic. As stated in the previous point, if a custom directive is used in multiple subgraphs to resolve a particular field, you should define the same directive with the same logic across subgraphs. _Composition does not detect or warn about such inconsistencies._
+- The composition process treats [executable (client-side)](/federation/federated-types/composition/#executable-directives) and [type system (server-side) directives](/federation/federated-types/composition/#type-system-directives) differently:
+  - An executable directive is composed into the supergraph schema if:
```

Comment:
nit: may be worth linking to https://www.apollographql.com/docs/federation/federated-types/composition#executable-directives to have a single location

Reply:
I can see why it would be nice to link both executable and type system to one place since they are so close together in that page, but for accuracy's sake will keep them apart here.

Reply:
Oh maybe you meant just link to that doc rather than restating its contents here. For a better UX would prefer to not make folks open another page.

Reply:
ah yeah, second point but agree having a sole location would be nice. alternatively we could have the above section link here